grails-app/views/species/tabs/_overview.gsp

```diff
@@ -1,6 +1,6 @@
 <section class="tab-pane active" id="tab-overview" role="tabpanel">
     <div class="row taxon-row">
-        <div class="col-md-5">
+        <div class="col-12 col-md-5">
```

Comment:
How does that wrok together?

Reply:
previously the map and list looked retarded